src/dashboard/events/cd-dashboard-upcoming-event.vue

```diff
@@ -138,18 +138,21 @@
   @import "../../common/variables";
   @import "../../common/styles/cd-filler-loading";
 
+  @wrap: 300px;
   .cd-dashboard-upcoming-event {
     margin: 32px 0;
 
     &__content {
       background: @cd-white;
       display: flex;
       padding: 0 20px;
+      flex-wrap: wrap;
```

Comment:
I thought we were leaving flex-wrap out of it now and just letting the breakpoint do it's thing?

Reply:
Yep, my bad, pre-walk rush :>